app/src/androidTest/java/net/skyscanner/backpack/meta/StoriesTests.kt

```diff
@@ -18,28 +18,32 @@
 
 package net.skyscanner.backpack.meta
 
-import net.skyscanner.backpack.demo.meta.Story
-import net.skyscanner.backpack.demo.meta.all
+import net.skyscanner.backpack.demo.meta.StoriesRepository
 import org.junit.Assert.assertTrue
-import org.junit.Ignore
 import org.junit.Test
 
-@Ignore("These tests are ignored because we need to migrate the stories first")
 class StoriesTests {
```

Comment:
now that we've got the right stories can you make these tests a bit more useful - ie test the expected properties are set etc, rather than just "they exist"

Reply:
But we do test the properties – e.g. whether it's a compose vs view story, as well as component definitions

Reply:
these still just test for "is there any story that has x" rather than "is this story the expected result"?